nntrainer/src/conv2d_layer.cpp

```diff
@@ -152,7 +155,7 @@ Tensor Conv2DLayer::backwarding(Tensor derivative, int iteration) {
   for (unsigned int b = 0; b < input_dim.batch(); ++b) {
     Tensor in_padded = zero_pad(b, input, padding);
     TensorDim p_dim(1, 1, in_padded.height(), in_padded.width());
-
+    float sum = 0.0;
     /// @fixme: #280
     for (unsigned int i = 0; i < filter_size; i++) {
       Tensor &delK = paramsAt(i).grad;
```

Comment:
delK should be initialized with `setZero` here to remove the effect of training from previous iteration.

Reply:
right. thanks.